packages/js/product-editor/src/components/variations-table/update-stock-menu-item/update-stock-menu-item.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { TRACKS_SOURCE } from '../../../constants';
+import { handlePrompt } from '../../../utils/handle-prompt';
+import { VariationActionsMenuItemProps } from '../types';
+
+export function UpdateStockMenuItem( {
+	selection,
+	onChange,
+	onClose,
+}: VariationActionsMenuItemProps ) {
+	return (
+		<MenuItem
+			onClick={ () => {
+				const ids = Array.isArray( selection )
+					? selection.map( ( { id } ) => id )
+					: selection.id;
+
+				recordEvent( 'product_variations_menu_inventory_select', {
+					source: TRACKS_SOURCE,
+					action: 'stock_quantity_set',
+					variation_id: ids,
```

Comment:
I wonder if we should also add an extra prop called `multi_selection` or something like that, that is true or false depending if `selection` is an array or not.
That will give a clearer differentiator for where the track was triggered, instead of looking at `variation_id` only.

Reply:
Yep, it's true, I forced the variation_id to be one or many. Let's see what @AnnaMag wants to day here.
Do we need a different event for multiple variations or just one while we send different args depending on how many variations are being modified?

I'm gonna merge this and tackle any suggestion made here in the next issue

Reply:
@mdperez86 
Ideally, the props would be non-redundant. That is, if we can easily infer a value of X based on other props, it is ok to skip it.  In this case, _multi_selection_ is easily parsed in SQL. I vote to leave it as-is.